uaa_bot/bot.py

```diff
@@ -115,3 +115,56 @@ def notify_deactivation_in_10_days(self) -> dict:
             80, "Account of deactivations in 10 days", "account_expires_in_10_days"
         )
         return summary
+
+    def get_all_user_last_logon(
+        self,
+        days_ago: int = 0,
+        days_range: int = 365,
+        summary_title: str = "List Users",
+        start_of_day: int = None,
+        end_of_day: int = None,
+        params: dict = {},
+    ) -> dict:
+        """
+        Gets list of users and their last logon info
+        """
+        users = []
+        uaac = UAAClient(uaa_config=self.uaa_config)
+        uaac.authenticate()
+        if start_of_day and end_of_day:
+            response = uaac.list_users_last_logon(
+                start_of_day=start_of_day, end_of_day=end_of_day, params=params
+            )
+        else:
+            response = uaac.list_users_last_logon(
+                days_ago=days_ago, days_range=days_range, params=params
+            )
+        resources = response.get("resources")
+
+        # Get user with their last logon info
+        for user in resources:
+            user_email = user.get("userName")
+            user_guid = user.get("id")
+            user_last_logon = time.ctime(user.get("lastLogonTime") / 1000)
+            user_active = user.get("active")
+            users.append(
```

Comment:
[request] Would it be possible for us to just get the API fields exactly as they appear in the UAA documentation?  Although that means a little more work for us to have to format the dates and whatnot, I think that will be helpful in the future if there's confusion about how particular fields map to the UAA API.  So this might look something more like:

```json
{
    "userName" : user.get("userName"),
    "id": user.get("id"),
    "lastLogonTime": user.get("lastLogonTime"),
    "active": user.get("active")
}
```

Particularly, I'm worried that the "user_email" field will be confusing to someone down the road in case the username is not an email (like in the dev environment).  If you wanted to still give us a pretty date instead of the epoch, I am certainly not going to turn you down, we just might want an indicator of what the timezone is (maybe in a top level field or within the date format itself)?